packages/react-components/react-portal-compat-context/package.json

```diff
@@ -16,7 +16,7 @@
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
-    "lint": "just-scripts lint",
+    "lint": "just-scripts lint && just-scripts lint-imports",
```

Comment:
we'll need to implement this as well to migration generator , adding for tracking https://github.com/microsoft/fluentui/issues/23930